Gemfile

```diff
@@ -188,7 +188,7 @@ gem 'highline', '~> 1.6.21'
 
 gem 'honeybadger', '>= 4.5.6' # error monitoring
 
-gem 'newrelic_rpm', '~> 6.14.0', group: [:staging, :development, :production] # perf/error/etc monitoring
+gem 'newrelic_rpm', '~> 6.14.0', require: false, group: [:staging, :development, :production] # perf/error/etc monitoring
```

Comment:
https://github.com/code-dot-org/code-dot-org/blob/d9caabe9ac007e1c99ba398db5c6c99ce4221e38/dashboard/config/application.rb#L212-L214

Reply:
This feels like a potentially very broad change; I'm definitely not opposed to us exploring it, but I am opposed to us trying to do it as a small piece of a PR that's largely focused elsewhere.

Based on my manual testing of this script, I think it's plenty performant without this optimization; I vote we leave as-is for now, and consider this for a follow-up

Reply:
https://github.com/code-dot-org/code-dot-org/pull/59962/commits/1ad136ecf01555113a5c2a25836ff9ea92513e7d